vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -1612,7 +1612,12 @@ EXPORT(bool, sceGxmProgramIsDiscardUsed, const SceGxmProgram *program) {
 EXPORT(bool, sceGxmProgramIsEquivalent, const SceGxmProgram *programA, const SceGxmProgram *programB) {
     assert(programA);
     assert(programB);
-    return UNIMPLEMENTED();
+
+    if (!programA || !programB)
+        LOG_ERROR(SCE_GXM_ERROR_INVALID_POINTER);
+
+    STUBBED("Compare programs, not output");
+    return programA == programB;
```

Comment:
```suggestion
    return (programA->size == programB->size) && (memcmp(programA, programB, programA->size) == 0);
```

Reply:
confirmed, is good implement like that, and no stub

Reply:
both programs can be different primitively and have same output iirc

Reply:
No, hete with yhis code you compare size and shader in memory directy